source/release-notes/backend.txt

```diff
@@ -13,6 +13,17 @@ Atlas App Services Changelog
 2024 Releases
 -------------
 
+.. _backend_20240207:
+
+7 February 2024 Release
+~~~~~~~~~~~~~~~~~~~~~~~
+
+- Atlas Data Services log pages now use your timezone of preference.
+
+- Added support for dictionaries in Dart object model generation.
+
+- Apps linked to serverless datasources cannot enable sync. 
```

Comment:
this one's confusing to me - what actually changed? I thought [this was already the case](https://www.mongodb.com/docs/atlas/app-services/mongodb/#:~:text=You%20cannot%20use%20a%20serverless%20instance%20as%20your%20app%27s%20Device%20Sync%20cluster.) ...or were they still technically able to when they shouldn't have been?
```suggestion
- Apps linked to serverless datasources can no longer enable Device Sync <because reasons...>. 
```

Reply:
Good catch, it is a bug fix. Updated wording!